packages/truffle-debugger/lib/data/sagas/index.js

```diff
@@ -62,35 +62,37 @@ function* tickSaga() {
 
   switch (node.nodeType) {
     case "FunctionDefinition":
-      parameters = node.parameters.parameters.map(
-        (p, i) => `${pointer}/parameters/parameters/${i}`
-      );
+      let parameters = node.parameters.parameters;
+      //note that we do *not* include return parameters, since those are
+      //handled by the VariableDeclaration case (no, I don't know why it
+      //works out that way)
```

Comment:
huh that's new?

Reply:
or at least news

Reply:
Well, it can't be new -- if it were, the old code, which put them below rather than above the input parameters -- wouldn't have ever worked...